.github/workflows/ci.yml

```
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [16.x, 15.x, 14.x, 13.x, 12.x, 10.x, "10.12.0"]
+        node: [16.x, 14.x, 12.x, "12.22.0"]
```

Shouldn't we test at least against the minimum version of each major Node version we support? 🤔

```suggestion
        node: [16, "16.0.0", 14, 14.17, 12, 12.22]
```

---

I don't feel strong, as it should have been covered by eslint-plugin-node's `no-unsupported-features/*`:
* [node/no-unsupported-features/es-builtins](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features/es-builtins.md)
* [node/no-unsupported-features/es-syntax](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features/es-syntax.md)
* [node/no-unsupported-features/node-builtins](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unsupported-features/node-builtins.md)

---

Historically, we've tested the absolute minimum version but not the minimum of each major to reduce the number of CI builds. If we ever accidentally introduce something that passes all the builds we have but fails on an earlier supported version, we'd then introduce the additional builds for each minimum major version. Thankfully we haven't needed to do that yet.

---

Marking as resolved.